### PR TITLE
Add stations from form

### PR DESCRIPTION
**Pull request by unsign3d**
This PR add a small form that saves the stations into localstorage.

For debug purposes we can add everything there

### CODE REVIEW

**Comment by jduthon** [comment]
If we wanted this to be a tad more reactive we could avoid the page reloading by having some oberserver on the form, but I think for now it's good enough.

What happens though if you enter an incorrect station name?
And maybe we can have a "Clean stations" button in case things go south?

**Comment by unsign3d** [comment]
sure thing

**Comment by unsign3d** [comment]
```What happens though if you enter an incorrect station name?```
As we saw yesterday it's a search, so I would put the autocomplete in a followup since IMHO it's not a deal breaker, I will do the rest of it in the next couple of days

**Comment by jduthon** [comment]
@unsign3d yeah if adding a wrong station name doesn't lead to a crash happy to add it like this for now :)
If it leads to a crash we can maybe as a first step handle the crash and not add the station?
I'd say we don't even need the UI to reflect that for now, we can have just a `console.warn`

**Comment by unsign3d** [comment]
it just ignores what you wrote apparently

**Comment by jduthon** [comment]
Ok that's fine :)

**Comment by unsign3d** [comment]
I needed to do some refactoring for the inputs, now all the inputs are rendered in one function and you can use that function to redraw.

**Comment by unsign3d** [comment]
we also should think about a better way to handle the observers because we need to "restart them" every time that we dinamically change the inputs

File: input/vbb/drawStations.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { makeIndirectFetch } from "../../fn-helpers";
+
+
+const pollDepartures = stationId =>
+poll(2000, makeIndirectFetch(fetchDepartures)({ stationId }));
+
+
+export const drawStations = (app) => {
+  if (window.stops) { 
+    window.stops.unsubscribe();
```
Comment (jduthon) [inline input/vbb/drawStations.js:21]: If you want this to work you should update the code on line 41 from `forEach` to `map` and here you should do `window.stops.forEach(stop => stop.unsubscribe())`
I'd also recommend naming the `stops` either `stops$` or `stopsObservable` so it's a bit clearer what is in there.

Overall though I don't think the observable code should end up in `drawStations`.
Main point behind this being the observables part is mostly input, the drawing part is output.

File: input/vbb/stations/form.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+const addStation = (app) => () => {
+  const station = document.getElementById("station_input").value
+  console.log(`added station: ${station} to memory`)
+  addStationApi(station)
+  redraw(app)
+}
+
+const redraw = (app) => {
```
Comment (jduthon) [inline input/vbb/stations/form.js:38]: As we are using rx, I'd be happier not to have this function, and instead make an observable `stationNames$` or `stationNamesObservable` and link that to the rest of the pipeline :)

**Pull request closed without merging**